main.js

```diff
@@ -0,0 +1,12 @@
+var parser = require('./parser.js').parser,
+    makeXPathModels = require('./models.js').makeXPathModels;
+
+parser.makeXPathModels = makeXPathModels;
+
+parser.setXPathModels = function(models) {
+    parser.yy.xpathmodels = models;
```

Comment:
Yeah. I did it this way so that the library would still provide a default xpath implementation that doesn't use hashtags when included.
This does allow a user to do:

```
var newParser = new xpath.Parser();
newParser.yy.xpathmodels = xpath.makeXPathModels(...);
newParser.parse('#this/can/use/different/xpathmodels');
```